test/native/lib/TestModule.re

```diff
@@ -0,0 +1,29 @@
+open Migrate_parsetree;
+open Ast_410;
+open Setup;
+
+open Ppxlib.Ast_builder.Make({
+       let loc = Location.none;
+     });
+
+let compare = (result, expected, {expect, _}) => {
+  let result = Pprintast.string_of_structure([result]);
+  let expected = Pprintast.string_of_structure([expected]);
+  expect.string(result).toEqual(expected);
+};
+
+describe("transform module ppx", ({test, _}) => {
```

Comment:
This message is too generic, but I couldn't think about anything else, at least now we have a place to test more generated code, maybe even testing the complete module

Reply:
It's good enough